conf/application.conf

```diff
@@ -59,3 +59,13 @@ logger.play=INFO
 
 # Logger provided to your application:
 logger.application=DEBUG
+
+# Hub configuration
+# ~~~~~
+
+# Email address of site administrator
+hub.admin.email="rrodgers@mit.edu"
```

Comment:
You may want to make this an ENV var as well to allow future forkers to deploy without your email embedded.